src/structures/BaseCommandInteraction.js

```diff
@@ -169,6 +180,10 @@ class BaseCommandInteraction extends Interaction {
 
       const role = resolved.roles?.[option.value];
       if (role) result.role = this.guild?.roles._add(role) ?? role;
+
+      const attachment = resolved.attachments?.[option.value];
+      const patched = new MessageAttachment(attachment.url, attachment.filename, attachment);
```

Comment:
this will error if attachment is undefined

Reply:
good catch! thanks!